src/steps/elements.tsx

```diff
@@ -76,6 +77,11 @@ const StepButton: React.FC<StepButtonProps> = ({
           color: danger ? COLORS.danger5 : COLORS.neutral90,
           boxShadow: "0 0 8px var(--shadow-color)",
           ...danger && { backgroundColor: COLORS.danger1 },
+          ...isHighContrast && {
+            outline: `2px solid ${danger ? COLORS.danger5 : COLORS.accent4}`,
+            borderColor: "transparent",
+            boxShadow: "none",
```

Comment:
I think it would be better to have the `boxShadow` declaration above be `boxShadow: isHighContrast ? "none" : ...` instead of overwriting it here.